lib/msal-node/package.json

```diff
@@ -28,9 +30,10 @@
     "start": "tsdx watch",
     "build": "tsdx build",
     "test": "tsdx test .*.spec.*",
-    "test:watch": "tsdx test .*.spec.* --watch",
-    "test:coverage": "tsdx test .*.spec.* --coverage",
-    "lint": "tsdx lint",
+    "test:n": "tsdx test",
+    "test:w": "tsdx test .*.spec.* --watch",
+    "test:c": "tsdx test .*.spec.* --coverage",
```

Comment:
I would argue to spell out the words, e.g. `watch`, `coverage`, and then have the main `npm test` command run the default test command.

Reply:
okay! No preference, this was for typing convenience. I wonder if we can create aliases for commands.

Reply:
Done.